src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsBtfBlocking.java

```diff
@@ -110,7 +112,28 @@ public void adsAtfOnTabletOasis(String wikiName, String article, boolean isWgVar
                          AdsContent.PREFOOTER_RIGHT);
   }
 
-  @RelatedIssue(issueID = "ADEN-3761")
+  @Execute(mockAds = "true")
```

Comment:
have you considered using:
`
@InBrowser(
    emulator = Emulator.GOOGLE_NEXUS_5,
    browser = Browser.CHROME
)
` ?